package.json

```diff
@@ -24,22 +24,35 @@
     "npm": ">= 3.x"
   },
   "bugs": {
-    "url": "https://github.com/xsellier/joi4express/issues"
+    "url": "https://gitlab.com/xsellier/joi4express/issues"
   },
-  "homepage": "https://github.com/xsellier/joi4express#readme",
+  "homepage": "https://gitlab.com/xsellier/joi4express#readme",
   "peerDependencies": {
     "joi": ">= 10.x.x",
     "boom": ">= 5.x.x"
   },
   "devDependencies": {
     "boom": ">= 5.x.x",
     "chai": "^4.0.2",
-    "dirty-chai": "^1.2.2",
+    "cross-env": "^5.2.0",
```

Comment:
I'm not sure why we need that?

Reply:
I mean it won't run on bare shell windows, but you can always install cygwin or something like that to develop on windows.